allennlp/data/dataset_readers/__init__.py

```diff
@@ -16,3 +16,5 @@
 from allennlp.data.dataset_readers.seq2seq import Seq2SeqDatasetReader
 from allennlp.data.dataset_readers.coreference_resolution import ConllCorefReader, WinobiasReader
 from allennlp.data.dataset_readers.penn_tree_bank import PennTreeBankConstituencySpanDatasetReader
+from allennlp.data.dataset_readers.stanford_sentiment_tree_bank import (
```

Comment:
`__init__.py` files deep in the hierarchy are a case where I'd be just fine with disabling pylint's `line-too-long` check.  No strong preference if you prefer it this way, though.